backend/src/litigation_api/ingestion/pipeline.py

```
@@ -0,0 +1,102 @@
+                raw_document = CRLCADocument(**raw_document_dict)
+
+                # Documents need a reference to a docket entry.
+                # In CRLCA v2.1 API, we usually just associate documents to the main case/docket,
+                # but if we don't have a specific docket link, we use the first docket or a dummy one.
+                entry_id = domain_dockets[0].entry_id if domain_dockets else f"crlca_docket_unknown_{case_id}"
+
+                domain_doc = normalize_document(raw_document, domain_case.case_id, entry_id)
+
+                # Check document type support
+                if domain_doc.document_type in SUPPORTED_DOCUMENT_TYPES and raw_document.file:
+                    logger.info(f"Downloading supported document: {domain_doc.doc_id}")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)